src/resources/swagger.yaml

```diff
@@ -55,10 +55,10 @@ paths:
         '200':
           description: 'Sample registration and Clinical Submission disabled state has been set'
 
-  /submission/schema:
+  /dictionary/migration/run:
     get:
```

Comment:
the get here is incorrect right ?

Reply:
before it was /dictionary/ and returns to you the current dictionary

Reply:
your right it should be different endpoint will fix